.github/workflows/pr-testing-build.yml

```diff
@@ -0,0 +1,32 @@
+# avoid replacing current release artifacts keeping pr testing build
+
+name: "PR build testing"
+on:
+    push:
+      branches-ignore:
+          - main   # excludes master
+env:
+    CI: false
+jobs:
+    publish-tauri:
+        strategy:
+            fail-fast: false
+            matrix:
+                platform: [macos-latest, ubuntu-20.04, windows-latest]
+        runs-on: ${{ matrix.platform }}
+        steps:
+            - uses: actions/checkout@v3
+            - name: setup node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 16
+            - name: install Rust stable
+              uses: dtolnay/rust-toolchain@stable
+            - name: install dependencies (ubuntu only)
+              if: matrix.platform == 'ubuntu-20.04'
+              run: |
+                  sudo apt-get update
+                  sudo apt-get install -y libgtk-3-dev libwebkit2gtk-4.0-dev libappindicator3-dev librsvg2-dev patchelf
+            - name: install app dependencies and build it
+              run: yarn && yarn build
```

Comment:
Same here, we probably want `yarn tauri:build` to build the whole app, not just the frontend

```suggestion
              run: yarn && yarn tauri:build
```